### PR TITLE
[Playground] notebooks for ingestion suitable for playground

### PR DESCRIPTION
**Pull request by joemcelroy**
Adding four notebooks. All using LangChain tooling for loading, chunking and ingesting into Elasticsearch. All use ELSER as the embedding model:
- PDF ingestion
- Webpage ingestion
- JSON ingestion

### CODE REVIEW

File: notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Website load, Extraction and Ingest with ELSER Example\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/ingestion/website-chunking-ingest.ipynb)\n",
+    "\n",
+    "This workbook demonstrates how to load a website, create passages and ingest into Elasticsearch. \n",
+    "\n",
+    "In this example we will:\n",
+    "- load the website using unstructured\n",
```
Comment (leemthompo) [inline notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb:13]:
```suggestion
    "- load the website using the [`unstructured` package](https://python.langchain.com/docs/integrations/providers/unstructured/)\n",
```

Comment (leemthompo) [reply notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb:13]: reads a little weird if you're not familiar with langchain

File: notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb [hunk 2]
```diff
@@ -0,0 +1,241 @@
+{
```
Comment (smith-kyle) [inline notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb:1]: _Commented on notebook `notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb` Cell 5 Line 1_
```markdown
## Deploying ELSER
```
test

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/232)._

File: notebooks/ingestion-and-chunking/json-chunking-ingest.ipynb [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip is available: \u001b[0m\u001b[31;49m23.3.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m24.0\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install -qU  pypdf langchain_community langchain elasticsearch tiktoken langchain-elasticsearch jq"
```
Comment (miguelgrinberg) [inline notebooks/ingestion-and-chunking/json-chunking-ingest.ipynb:42]: pypdf is unused in this notebook

File: notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m A new release of pip is available: \u001b[0m\u001b[31;49m23.3.1\u001b[0m\u001b[39;49m -> \u001b[0m\u001b[32;49m24.0\u001b[0m\n",
+      "\u001b[1m[\u001b[0m\u001b[34;49mnotice\u001b[0m\u001b[1;39;49m]\u001b[0m\u001b[39;49m To update, run: \u001b[0m\u001b[32;49mpip install --upgrade pip\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install -qU  pypdf langchain_community langchain elasticsearch tiktoken langchain-elasticsearch unstructured"
```
Comment (miguelgrinberg) [inline notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb:32]: Here too, pypdf can be removed.

**Review by miguelgrinberg — APPROVED** [review]
Just a couple of unused imports, but looks good. If you end up keeping the notebooks excluded from testing then the `_nbtest.teardown.ipynb` file can be removed.

**Pull request merged**